src/cli/javascript/js_request.rs

```diff
@@ -9,9 +9,7 @@ use rquickjs::{FromJs, IntoJs};
 use serde::{Deserialize, Serialize};
 
 use crate::core::is_default;
-
-#[derive(Debug)]
-pub struct JsRequest(reqwest::Request);
```

Comment:
Keep the JsRequest as it is. Move it to core, but drop the `FromJs` and `IntoJs` from core. Move those implementations to `cli::javascript::worker`